examples/face_landmarker/ios/FaceLandmarker/Helper/FaceLandmarkerHelper.swift

```diff
@@ -0,0 +1,139 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksVision
+import AVFoundation
+
+protocol FaceLandmarkerHelperDelegate: AnyObject {
+  func faceLandmarkerHelper(_ faceLandmarkerHelper: FaceLandmarkerHelper,
+                             didFinishDetection result: ResultBundle?,
+                             error: Error?)
+}
+
+class FaceLandmarkerHelper: NSObject {
+
+  weak var delegate: FaceLandmarkerHelperDelegate?
+  var faceLandmarker: FaceLandmarker?
+
+  init(modelPath: String?, numFaces: Int, minFaceDetectionConfidence: Float, minFacePresenceConfidence: Float, minTrackingConfidence: Float, runningModel: RunningMode, delegate: FaceLandmarkerHelperDelegate?) {
+    super.init()
+    guard let modelPath = modelPath else { return }
+    let faceLandmarkerOptions = FaceLandmarkerOptions()
+    faceLandmarkerOptions.runningMode = runningModel
+    faceLandmarkerOptions.numFaces = numFaces
+    faceLandmarkerOptions.minFaceDetectionConfidence = minFaceDetectionConfidence
+    faceLandmarkerOptions.minFacePresenceConfidence = minFacePresenceConfidence
+    faceLandmarkerOptions.minTrackingConfidence = minTrackingConfidence
+    faceLandmarkerOptions.faceLandmarkerLiveStreamDelegate = runningModel == .liveStream ? self : nil
+    self.delegate = delegate
+    faceLandmarkerOptions.baseOptions.modelAssetPath = modelPath
+    do {
+      faceLandmarker = try FaceLandmarker(options: faceLandmarkerOptions)
+    } catch {
+      print(error)
+    }
+  }
+
+  /**
+   This method return FaceLandmarkerResult and infrenceTime when receive an image
+   **/
+  func detect(image: UIImage) -> ResultBundle? {
+    guard let mpImage = try? MPImage(uiImage: image) else { return nil }
+    do {
+      let startDate = Date()
+      let result = try faceLandmarker?.detect(image: mpImage)
+      let inferenceTime = Date().timeIntervalSince(startDate) * 1000
+      return ResultBundle(inferenceTime: inferenceTime, faceLandmarkerResults: [result])
+    } catch {
+      print(error)
+      return nil
+    }
+  }
+
+  /**
+   This method return FaceLandmarkerResult and infrenceTime when receive videoFrame
+   **/
+  func detectAsync(videoFrame: CMSampleBuffer, orientation: UIImage.Orientation, timeStamps: Int) {
+    guard let faceLandmarker = faceLandmarker,
+          let image = try? MPImage(sampleBuffer: videoFrame, orientation: orientation) else { return }
+    do {
+      try faceLandmarker.detectAsync(image: image, timestampInMilliseconds: timeStamps)
+    } catch {
+      print(error)
+    }
+  }
+
+  /**
+   This method return FaceLandmarkerResults and infrenceTime when receive videoUrl and inferenceIntervalMs
+   **/
+  func detectVideoFile(url: URL, inferenceIntervalMs: Double) async -> ResultBundle? {
```

Comment:
Can we take each major step of this function and space it away from the others, then add a comment highlighting what's going on? Basically make it clearer for someone unfamiliar with how the app works to come in and understand it in smaller pieces.